### PR TITLE
Shutdown Lighty.io modules with synchronous shutdown

### PR DESCRIPTION
**Pull request by ihrasko**
Shutdown Lighty.io modules with synchronous shutdown method thus removing a lot of duplicate code.

### CODE REVIEW

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/main/java/io/lighty/applications/rnc/module/RncLightyModule.java [hunk 1]
```diff
@@ -171,22 +171,20 @@ private void startAndWaitLightyModule(final LightyModule lightyModule) throws Rn
...
-        if (this.rncModuleConfig.getAaaConfig().isEnableAAA()
-                && this.aaaLighty != null && !stopAndWaitLightyModule(this.aaaLighty)) {
-            success = false;
+        if (this.rncModuleConfig.getAaaConfig().isEnableAAA() && this.aaaLighty != null) {
+            success = aaaLighty.shutdown(lightyModuleTimeout, DEFAULT_LIGHTY_MODULE_TIME_UNIT);
         }
-        if (this.lightyRestconf != null && !stopAndWaitLightyModule(this.lightyRestconf)) {
-            success = false;
+        if (this.lightyRestconf != null) {
+            success = lightyRestconf.shutdown(lightyModuleTimeout, DEFAULT_LIGHTY_MODULE_TIME_UNIT);
```
Comment (PeterSuna) [inline lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/main/java/io/lighty/applications/rnc/module/RncLightyModule.java:181]: If previous shutdown fails and set `success` to false. This line can override the previous value to true. So at the end, success==true.

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/main/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModule.java [hunk 2]
```diff
@@ -155,14 +155,14 @@ public boolean close() {
...
 
-        if (this.lightyRestconf != null && !stopAndWaitLightyModule(this.lightyRestconf)) {
-            success = false;
+        if (this.lightyRestconf != null) {
+            success = lightyRestconf.shutdown(lightyModuleTimeout, DEFAULT_LIGHTY_MODULE_TIME_UNIT);
         }
-        if (this.lightyController != null && !stopAndWaitLightyModule(this.lightyController)) {
-            success = false;
+        if (this.lightyController != null) {
+            success = lightyController.shutdown(lightyModuleTimeout, DEFAULT_LIGHTY_MODULE_TIME_UNIT);
```
Comment (PeterSuna) [inline lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/main/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModule.java:162]: Overrides previous value.

**Review by PeterSuna — CHANGES_REQUESTED** [review]

File: lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/main/java/io/lighty/applications/rnc/module/RncLightyModule.java (hunk 1 above)
Comment (ihrasko) [reply lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-module/src/main/java/io/lighty/applications/rnc/module/RncLightyModule.java]: Done

File: lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/main/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModule.java (hunk 2 above)
Comment (ihrasko) [reply lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app-module/src/main/java/io/lighty/applications/rcgnmi/module/RcGnmiAppModule.java]: Done

**Pull request merged**